modules/47_regipol/regiCarbonPrice/sets.gms

```diff
@@ -142,6 +142,16 @@ $ifthen.cm_implicitQttyTarget not "%cm_implicitQttyTarget%" == "off"
   );
 $endIf.cm_implicitQttyTarget
 
+Sets
```

Comment:
It is recommended to calculate this set automatically by using something like:
```
loop ( pe2se(entyPe,entySe,te),
enty_pe2se(entyPe,entySe) = YES;
);
```
You can use the set `sefe` as a reference on how to do it. 

I would also avoid using the name `enty_pe2se` as it is a calculated set and not a primal one. 
In a perfect World this set should be named `pe2se`, and the current `pe2se` set should be instead `pe2sete` or `pe2se2te`, but as this is not the case I would instead follow the same naming convention used already for the final energy set (`sefe`) that follows the same structure, and name it `pefe` instead.

This set should also be defined at the core as it is a general set that can be used in other places of the model to simplify other statements.

Reply:
done